pkg/storage/fs/cback/cback.go

```diff
@@ -0,0 +1,473 @@
+package cback
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/url"
+	"strconv"
+	"time"
+
+	provider "github.com/cs3org/go-cs3apis/cs3/storage/provider/v1beta1"
+	v1beta1 "github.com/cs3org/go-cs3apis/cs3/types/v1beta1"
+	ctxpkg "github.com/cs3org/reva/pkg/ctx"
+	"github.com/cs3org/reva/pkg/errtypes"
+	"github.com/cs3org/reva/pkg/mime"
+	"github.com/cs3org/reva/pkg/storage"
+	"github.com/cs3org/reva/pkg/storage/fs/registry"
+	"github.com/mitchellh/mapstructure"
+	"github.com/pkg/errors"
+)
+
+type cback struct {
+	conf *Options
+}
+
+func init() {
+	registry.Register("cback", New)
+}
+
+// New returns an implementation to the storage.FS interface that talks to
+// cback
+func New(m map[string]interface{}) (fs storage.FS, err error) {
+
+	c := &Options{}
+	if err = mapstructure.Decode(m, c); err != nil {
+		return nil, errors.Wrap(err, "Error Decoding Configuration")
+	}
+
+	// returns the storage.FS interface
+	return &cback{conf: c}, nil
+
+}
+
+func (fs *cback) GetMD(ctx context.Context, ref *provider.Reference, mdKeys []string) (*provider.ResourceInfo, error) {
+	var ssID, searchPath string
+
+	user, _ := ctxpkg.ContextGetUser(ctx)
+	UID, _ := ctxpkg.ContextGetUserID(ctx)
```

Comment:
no need for this. You can get the userID from the user struct

Reply:
Now using user.Id